### ISSUE TITLE
tasks

### ISSUE THREAD

**Issue by wang333333** [CharafeddineMechalikh/PureEdgeSim#15]
Hi:
Is it possible to edit tasks in this platform?Make these tasks relevant.
Looking forward to your reply. Thank you very much

**Comment by CharafeddineMechalikh** [comment]
I think I can, i was thinking of adopting iFogSim models, 

Can you explain more please? 
Thank you.

**Comment by wang333333** [comment]
> 
> 
> I was thinking of adopting iFogSim application model,
> 
> However, i'm not sure if i understand what you mean, so, can you explain more please?
> Thank you.

What I mean is a complex task can be divided into several simple tasks, the tasks generated by the task generator can be linked to each other, or can be represented as a directed acyclic graph, unloaded to the server in the form of a task chain.
Thank you for your reply

**Comment by CharafeddineMechalikh** [comment]
Yes that's what I expected, I was thinking of that, however it is challenging to do it on a large scale. Besides I'm so busy these days. So, currently, I think it is better for you to use iFogSim for that type of applications modeling.

**Comment by wang333333** [comment]
Ok, thank you for your reply

**Comment by aatesh123** [comment]
i have assigned task to implement fuzzy randomized load balancing algorithm  on cloudsim ?
can anyone help me how I can start and from i get resource to implement this task?
